datafusion/Cargo.toml

```diff
@@ -64,7 +64,7 @@ pin-project-lite= "^0.2.7"
 tokio = { version = "1.0", features = ["macros", "rt", "rt-multi-thread", "sync", "fs"] }
 tokio-stream = "0.1"
 log = "^0.4"
-md-5 = { version = "^0.9.1", optional = true }
+md-5 = { version = "^0.10.0", optional = true }
```

Comment:
I bet md5/blake/sha2 need to be updated all together 🤔 

Any chance you have a moment to try that @xudong963  (so we can clear the backlog of dependabot updates?)

Reply:
Yeah, let me try